src/simplewallet/simplewallet.cpp

```diff
@@ -8786,6 +8786,8 @@ bool simple_wallet::rescan_blockchain(const std::vector<std::string> &args_)
     }
   }
 
+  m_in_manual_refresh.store(true, std::memory_order_relaxed);
```

Comment:
You may want `acquire` semantics here (the default for store), if the goal is to post information to another thread. The compiler+hardware is allowed to reorder the value change "after" this line from the perspective of another  thread.

Reply:
It's the same thread that'll read it, in a callback (but thanks for the note, I'm not familiar with these and always have to look at the docs).